README.md

```diff
@@ -1449,6 +1450,72 @@ $car->dump();
 
 **[⬆ back to top](#table-of-contents)**
 
+### Prefer final classes
+
+The `final` should be used whenever possible:
+
+1. It prevents uncontrolled inheritance chain.
+2. It encourages [composition](#prefer-composition-over-inheritance).
+3. It encourages the [Single Responsibility Pattern](#single-responsibility-principle-srp).
+4. It encourages developers to use your public methods instead of extending the class to get access on protected ones.
+5. It allows you to change your code without any break of applications that use your class.
+
+The only condition is that your class should implement an interface and no other public methods are defined.
+
+**Bad:**
+
+```php
+final class Car
+{
+    private $color;
+    
+    public function __construct($color)
+    {
+        $this->color = $color;
+    }
+    
+    /**
+     * @return string The color of the vehicule
+     */
+    public function getColor() 
+    {
+        return $this->color;
+    }
+}
+```
+
+**Good:**
+
+```php
+interface Vehicule
+{
+    /**
+     * @return string The color of the vehicule
+     */
+    public function getColor();
+}
+
+final class Car interface Vehicule
```

Comment:
This should be `final class Car implements Vehicule`, right? And Vehicle doesn't have an extra U before the l.

Reply:
Correct